tools/clang/test/HLSLFileCheck/validation/bitfield_swizzle.hlsl

```diff
@@ -0,0 +1,31 @@
+// RUN: %dxc -T ps_6_6 -HV 2021 %s | FileCheck %s
```

Comment:
This isn't a validation test. It shouldn't be in this directory. There is an hlsl/operators/swizzle/ subdir or else there is an existing bitfields test in hlsl/types/struct/